plugins/table.md

```diff
@@ -409,6 +409,8 @@ tinymce.init({
 | Name | Arguments | Description |
 |------| ------| ----------- |
 | insertTable | columns: number, rows: number | Insert a table with the given number of columns and rows at the current cursor location |
+| getClipboardCols |  | Returns the data for any columns cut or copied using `mceTableCutCol` or `mceTableCopyCol`. {{site.requires_5_4v}} |
+| setClipboardCols | cols: HTMLElement[] | Set the data to be used by `mceTablePasteColBefore` or `mceTablePasteColAfter` for pasting columns into a table. {{site.requires_5_4v}} |
```

Comment:
This should actually be `HTMLTableRowElement[]` instead of `HTMLElement[]` and likewise for the `setClipboardRows()` API.

Reply:
Fixed by https://github.com/tinymce/tinymce-docs/pull/1563/commits/60291ef6cfd53d51c46a56445b277f5fd9bd643c